tests/main/classic-ubuntu-core-transition/task.yaml

```diff
@@ -29,8 +31,7 @@ execute: |
 
     . "$TESTSLIB/pkgdb.sh"
     echo "Ensure core is gone and we have ubuntu-core instead"
-    dpkg --purge --force-depends snapd
-    apt-get install -f -y
```

Comment:
Hmm, I wonder what this thing is about, why did we `apt-get install` ... nothing here?